app/views/admin/creator_settings/new.html.erb

```diff
@@ -45,6 +45,9 @@
     <% end %>
 
     <%= inline_svg_tag("forem-background.svg", aria_hidden: true, class: "forem-background absolute bottom-0 right-0 hidden m:block") %>
+    <span id="accept-code-of-conduct-modal" class="hidden" aria-hidden="true"><%= render "pages/coc_text" %></span>
+    <span id="accept-terms-and-conditions-modal" class="hidden" aria-hidden="true"><%= render "pages/terms_text" %></span>
```

Comment:
```suggestion
    <span id="accept-code-of-conduct-modal" class="hidden"><%= render "pages/coc_text" %></span>
    <span id="accept-terms-and-conditions-modal" class="hidden"><%= render "pages/terms_text" %></span>
```

(suggestion / not blocking) - we don't need `aria-hidden` here, because the `class="hidden"` sets the CSS property `display: none` which hides content from assistive tech as well as visually 🙂